### ISSUE TITLE
Quest broken: Emberstrife and Seal of Ascension

### ISSUE THREAD

**Issue by Randalierer** [SunstriderEmu/BugTracker#3858]
**Describe the bug**

Emberstrife: The Quest given by Myranda the Hag provides you a "amulet of draconic subversion". When you wear it you are transformed into a dragon. Then you should be able to talk to emberstrife. But he is still hostile and dont talk to you.
Same quest npc; Seal of Ascension: You have to attack Emberstrife and bring him down to low life. Then you need to set the "unforged seal of ascension" on the ground and use the "orb of dragonic energy". While you mindcontrol Emberstrife with it you have to use one of the skills of the dragon to forge the seal. But this skill is missing in the pet bar.
So there is alot of trouble with this npc.

Broken Quest-NPC: https://classic.wowhead.com/npc=10321/emberstrife

Emberstrife Quest: https://classic.wowhead.com/quest=6570/emberstrife
Emberstrife Questgiver: https://classic.wowhead.com/npc=11872/myranda-the-hag
Emberstrife Questitem: https://classic.wowhead.com/item=16787/amulet-of-draconic-subversion

Seal of Ascension: https://classic.wowhead.com/quest=4743/seal-of-ascension
Seal of Ascension Questgiver: https://classic.wowhead.com/npc=10299/scarshield-infiltrator aka https://classic.wowhead.com/npc=10296/vaelan
Seal of Ascension Questitems: https://classic.wowhead.com/item=12323/unforged-seal-of-ascension
Seal of Ascension Questitems: https://classic.wowhead.com/item=12300/orb-of-draconic-energy

**Comment by michaelkittrell** [comment]
i tried this quest when the 2-3 weeks after the server was released.  I don't remember a skill being missing (i remember using the standard fire breath to forge tit), but breathing fire on the unforged seal did nothing.   I tried it about 15 different ways... in group, in a raid, solo, different points on the ground, far away from the spawn, but I could never trigger it to forge the seal and gave up.
